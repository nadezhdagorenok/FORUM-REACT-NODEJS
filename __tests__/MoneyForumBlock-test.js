"use strict";

import React from 'react';
import renderer from 'react-test-renderer';

import MoneyForumBlock from '../components/MoneyForumBlock.js';

test('render MoneyForumBlock', () => {

    const component = renderer.create(
        <MoneyForumBlock title="Currency exchanges, Deposites, Rates!" />
    );
    let componentTree = component.toJSON();
    expect(componentTree).toMatchSnapshot();
});



