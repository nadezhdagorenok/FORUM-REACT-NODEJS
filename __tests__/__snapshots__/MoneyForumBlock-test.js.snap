// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render MoneyForumBlock 1`] = `
<div
  className="preloader"
>
  <div
    className="loader"
  />
</div>
`;
